docs/help/operator.md

```diff
@@ -1029,3 +1031,117 @@ stringData:
   jolokiaUser: alice
   jolokiaPassword: password1
 ```
+
+## Using Cert-Manager certificates to configure brokers
+
+Note: this feature currently is experimental. Feed back is welcomed.
+
+[cert-manager](https://cert-manager.io/) adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.
+
+The operator provides options in the custom resource that utilizes cert-manager x509 certificates to configure SSL/TLS transports for brokers.
+
+For a certificate to be used by broker, it has to have the keystore configured in it(either `pkcs12` or `jks`). For example:
```

Comment:
the "it(" looks like a typo, maybe the brackets are unnecessary

Reply:
yeah, I'll delete it. thx